spec/controllers/lists_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+
+describe ListsController do
+
+  include Devise::TestHelpers
+
+  before do
+    @user = create(:user)
+    sign_in @user
+  end
+
+  describe '#create' do
+    it "creates 2 lists" do
+      post :create, list: { title: 'test', user: @user }
+      @list = List.last
+      expect(List.count).to eq(1)
+      expect(@list.title).to eq('test')
+      expect(response).to redirect_to(list_path(@list.id))
```

Comment:
Minor tweak possible, `list_path(@list.id)` can be simplified to `list_path(@list)`